rex/utilities/utilities.py

```diff
@@ -731,3 +732,35 @@ def slice_sites(shape, chunks, sites=None, chunks_per_slice=5):
         raise TypeError(msg)
 
     return slices
+
+
+def res_dist_threshold(lat_lons, tree=None, margin=1.05):
+    """
+    Estimate the distance between resource points. Calculated as half of the
+    diagonal between closest resource points, with desired extra margin
```

Comment:
We need to clarify this. The method in techmap was to find the maximum distance threshold that an exclusion pixel could be from a resource cell center which happens when the exclusion pixels is in the exact middle/intersection of four resource grid cells. However, this reads like its the actual distance between resource grid cells, which would be either the lateral distance or the diagonal distance, not HALF of the diagonal. Importantly, what is this used for besides the techmap?

Reply:
Its pretty much always used for cKDTree distance thresholding

Reply:
So far...

Reply:
Okay can we just modify the text "Estimate the distance between resource points"? Because that's not really true.

Reply:
"Distance threshold for nearest resource points. Calculated as half of the
diagonal between closest resource points, with desired extra margin"?

Reply:
or
"Distance threshold for nearest neighbor searches performed on resource points. Calculated as half of the
diagonal between closest resource points, with desired extra margin"

Reply:
Either is great! and much more accurate :)